### PR TITLE
Francisco torres - week1 challenges

### PR DESCRIPTION
**Pull request by ghost**
This branch contains all files to solve the challenges for the first week. The challenges are into its own path with the challenge name as a reference.

### CODE REVIEW

**Comment by ghost** [comment]
Commits completed..

File: Week1/francisco-torres/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/kata/AccountServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+        try {
+            Constructor<Account> c = Account.class.getDeclaredConstructor(String.class, int.class, AccountType.class);
+            c.setAccessible(true);
+            currentAccount = c.newInstance("Ana", 0, AccountType.CURRENT);
+            savingAccount = c.newInstance("Pedro", 0, AccountType.SAVING);
+        } catch (Exception e) {
```
Comment (estigma88) [inline Week1/francisco-torres/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/kata/AccountServiceImplTest.java:39]: In unit tests we should catch exceptions, in this case, let it throw. I

File: Week1/francisco-torres/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/kata/AccountServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        transferAmount = 2500;
+        initialBalanceCurrentAccount = 3000;
+        initialBalanceSavingAccount = 1500;
+
+        currentAccount.setBalance(initialBalanceCurrentAccount);
+        savingAccount.setBalance(initialBalanceSavingAccount);
+
+        try {
+            accountService.transferMoney(currentAccount,savingAccount,transferAmount);
+        } catch (Exception e) {
```
Comment (estigma88) [inline Week1/francisco-torres/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/kata/AccountServiceImplTest.java:60]: Same here

File: Week1/francisco-torres/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/kata/AccountServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        transferAmount = 400;
+        initialBalanceCurrentAccount = 100;
+        initialBalanceSavingAccount = 200;
+
+        currentAccount.setBalance(initialBalanceCurrentAccount);
+        savingAccount.setBalance(initialBalanceSavingAccount);
+
+        try {
+            accountService.transferMoney(currentAccount,savingAccount,transferAmount);
+        } catch (Exception e) {
```
Comment (estigma88) [inline Week1/francisco-torres/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/kata/AccountServiceImplTest.java:79]: Same here

**Review by estigma88 — CHANGES_REQUESTED** [review]

**Review by estigma88 — APPROVED** [review]
Good job. I let you some comments to take into account next time

**Pull request merged**